### ISSUE TITLE
Neither warning nor error announced when both are at cursor position

### ISSUE THREAD

**Issue by chrmarti** [microsoft/vscode#213668]
Testing #213344

I hear "warning" and "error" when only one of the two is at the cursor position, but I don't hear any of the two when there are both:
![Image](https://github.com/microsoft/vscode/assets/9205389/7d9760e3-d5a3-44e6-bc3b-6ab9b52f6770)

**Comment by chrmarti** [comment]
Not sure if this is a bug, but it doesn't read the warning or error message when only one is present, but just says "warning" or "error" and nothing more.

**Comment by meganrogge** [comment]
Closing as I haven't heard user reports about this
